packages/primitives/table/src/types.ts

```diff
@@ -1,8 +1,22 @@
 export type TTableBorderStyle = {
+  borderColor?: string,
+  borderTopWidth?: number,
+  borderBottomWidth?: number,
+  borderLeftWidth?: number,
+  borderRightWidth?: number,
+  borderStyle?: 'solid' | 'dotted' | 'dashed' | 'none',
+}
+
+export type TTableBorderStyleNative = {
   borderColor?: string,
   borderTopWidth?: number,
   borderBottomWidth?: number,
   borderLeftWidth?: number,
   borderRightWidth?: number,
   borderStyle?: 'solid' | 'dotted' | 'dashed',
 }
+
+export type TTableCellPosition = {
+  position?: 'fixed' | '-moz-initial' | 'inherit' | 'initial' | 'revert' | 'unset' | '-webkit-sticky' | 'absolute' | 'relative' | 'static' | 'sticky',
```

Comment:
I added all the options that TypeScript suggested, but maybe it's an overkill?  Maybe just `static | relative | absolute | sticky | fixed` (from mozilla docs) would be enough? What do you think @deepsweet ?

Reply:
types for Web and Native should be the same, like the smallest possible intersection of universal set of values. My vote is for `static | relative | absolute | sticky | fixed` (not sure about `sticky` though).

Reply:
Ok, thanks! I'll fix it up :)